src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Subscriptions/DependencySharedProjectsSubscriber.cs

```diff
@@ -185,41 +186,33 @@ private void SubscribeToConfiguredProject(IProjectSubscriptionService subscripti
             IEnumerable<string> addedSharedImportPaths = sharedFolderProjectPaths.Except(currentSharedImportNodePaths);
             foreach (string addedSharedImportPath in addedSharedImportPaths)
             {
-                IDependencyModel added = CreateDependencyModel(addedSharedImportPath, targetContext.TargetFramework, resolved: true);
+                IDependencyModel added = CreateDependencyModel(addedSharedImportPath, resolved: true);
```

Comment:
Why isn't TargetFramework required anymore? Shared imports aren't expected to vary per framework?

Reply:
`SharedProjectDependencyModel` (nor any other `DependencyModel` subclass) doesn't store a target framework.

These models are currently added to a `DependenciesRuleChangeContext` which provides the framework.

In general I'm trying to remove stuff that's provably unused (even if it might be useful one day) as I find it makes the code easier to understand. It's easier to add code than remove it.

I'm still getting my head around the layering here. These are good questions for me, cheers.